updater/main.go

```diff
@@ -74,6 +74,19 @@ func _main() error {
 		ec2:            ec2.New(sess, aws.NewConfig()),
 	}
 
+	family, err := taskDefFamily()
+	if err != nil {
+		return fmt.Errorf("Failed to parse updater task definition arn: %w", err)
+	}
```

Comment:
I don't think we need to require the task def to be populated here for the updater to run.  If there's an error, we can skip the call to `u.alreadyRunning` and proceed.  This PR is a best-effort approach; we're not using a more reliable method like holding a lock or lease.